src/main/java/org/osiam/resources/exceptions/OsiamExceptionHandler.java

```diff
@@ -36,10 +35,11 @@
 
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
+
 @ControllerAdvice
 public class OsiamExceptionHandler extends ResponseEntityExceptionHandler {
 
-    private static final Logger LOGGER = Logger.getLogger(OsiamExceptionHandler.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger(OsiamExceptionHandler.class);
```

Comment:
I'm afraid that I've got a different opinion here. I don't think it's a good idea to fill the source code with unnecessary visual clutter if there is a method provided by the used library that does this anyway. Especially for something that should be injected but can't because of Springs lack of knowledge about the injection point.

However I would be happy to come up with a pull request that drops the `class.getName()` from the two occurrences in the `auth-server`